keystone/rebaseline_multi.py

```diff
@@ -0,0 +1,173 @@
+import numpy
+import scipy.sparse as sparse
+from scipy.sparse.linalg import spsolve
+from spectral_cube import SpectralCube
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.optimize import least_squares as lsq
+import numpy.polynomial.legendre as legendre
+import pprocess
+import time
+import sys
+import skimage
+from keystone.baseline import mad1d, legendreLoss
+
+def get_mask(spectra, mask_percent=0.4):
+      	"""  
+ Returns a mask of channels to be used for baseline fitting.
+ Function calculates the standard deviation of a 31 pixel window
+ centred on each pixel. A percentage of the pixels with the lowest 
+ standard deviation for their window are then chosen for baseline fitting.    
+   
+ spectra = input spectra as numpy array
+ mask_percent = percentage of pixels to select for baseline fitting    
+       	""" 
+	spec_len = len(spectra)
+	sample = int(spec_len*mask_percent)
+	left = numpy.zeros(15)+numpy.std(spectra[0:31]) # For the first 15 entries, use first window
```

Comment:
`numpy` to `np` maybe.  
Should this be a `nanstd` throughout?

Reply:
That would be better. I was assuming all spectra with at least one non-nan value contained all real signals.